winpty/ptyprocess.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, pty):
         assert isinstance(pty, PTY)
         self.pty = pty
         self.pid = pty.pid
+        self.read_blocking = bool(os.environ.get('PYWINPTY_BLOCK', 1))
```

Comment:
Let's change this to

    bool(os.environ.get('PYWINPTY_BLOCK', True))

for clarity.

Reply:
Sorry, this simply needs to be

    os.environ.get('PYWINPTY_BLOCK', True)

Reply:
Sorry, forget about this. I think this is fine (I didn't follow the logic correctly).